screen.yaml

```
@@ -27,15 +27,32 @@ pipeline:
       expected-sha256: f04a39d00a0e5c7c86a55338808903082ad5df4d73df1a2fd3425976aed94971
       uri: https://ftp.gnu.org/gnu/screen/screen-${{package.version}}.tar.gz
 
+  # These patches are taken from the oss-security mailing list:
+  # https://www.openwall.com/lists/oss-security/2025/05/12/1
+  - uses: patch
+    with:
+      patches: |
+        0001-logfile-reintroduce-lf_secreopen-to-fix-CVE-2025-233.patch 0002-default-PTY-mode-apply-safe-default-mode-of-0620-to-.patch 0003-attacher.c-fix-bad-strncpy-which-can-lead-to-a-buffe.patch 0004-attacher.c-prevent-temporary-0666-mode-on-PTYs-to-fi.patch 0005-Avoid-file-existence-test-information-leaks-to-fix-C.patch 0006-socket.c-don-t-send-signals-with-root-privileges-to-.patch
```

```suggestion
        0001-logfile-reintroduce-lf_secreopen-to-fix-CVE-2025-233.patch
        0002-default-PTY-mode-apply-safe-default-mode-of-0620-to-.patch
        0003-attacher.c-fix-bad-strncpy-which-can-lead-to-a-buffe.patch
        0004-attacher.c-prevent-temporary-0666-mode-on-PTYs-to-fi.patch
        0005-Avoid-file-existence-test-information-leaks-to-fix-C.patch
        0006-socket.c-don-t-send-signals-with-root-privileges-to-.patch
```

(assuming our patch pipeline is capable of handling that...)

---

Huh, I thought I'd tried that in the past and it hadn't worked, but... it does. Fixed!